app/components/Admin/Intake.tsx

```diff
@@ -71,28 +101,59 @@ const Intake: React.FC<IntakeProps> = ({ currentIntakeNumber, intake }) => {
   const { ccbcIntakeNumber, closeTimestamp, description, openTimestamp } =
     queryFragment;
 
-  const openDate = DateTime.fromISO(openTimestamp).toLocaleString(
+  const [archiveIntake] = useArchiveIntakeMutation();
+
+  const formattedOpenDate = DateTime.fromISO(openTimestamp).toLocaleString(
     DateTime.DATETIME_FULL
   );
 
-  const closeDate = DateTime.fromISO(closeTimestamp).toLocaleString(
+  const formattedCloseDate = DateTime.fromISO(closeTimestamp).toLocaleString(
     DateTime.DATETIME_FULL
   );
 
+  const currentDateTime = DateTime.now();
+  const endDateTime = DateTime.fromISO(closeTimestamp);
+  const isAllowedDelete = currentDateTime <= endDateTime;
```

Comment:
Should this be using the open timestamp instead of the close one, it is currently letting me delete in progress/open intakes. 

Something like this: 

` const startDateTime = DateTime.fromISO(openTimestamp);`
 ` const isAllowedDelete = currentDateTime <= startDateTime;`

Reply:
Ah yeah good catch, fixed!